region/maridia/inner-pink.json

```diff
@@ -2628,6 +2672,41 @@
                   ]
                 }
               ]
+            },
+            {
+              "id": 4,
+              "strats": [
+                {
+                  "name": "Halfie Climb Carry Shinecharge through Morph Tunnel to the Top",
+                  "notable": true,
+                  "requires": [
+                    "Morph",
+                    "Gravity",
+                    "canShinechargeMovementComplex",
+                    {"or": [
+                      {"canShineCharge": {
+                        "usedTiles": 33,
+                        "shinesparkFrames": 39,
+                        "excessShinesparkFrames": 4,
+                        "openEnd": 0
+                      }},
+                      {"and": [
+                        "canMidairShinespark",
+                        {"canShineCharge": {
+                          "usedTiles": 33,
+                          "shinesparkFrames": 33,
+                          "excessShinesparkFrames": 3,
```

Comment:
excess frames should be the same 👀

Reply:
This one is tricky. If youre jumping, I'm assuming youre diagonal sparking, so you hit the tile 1 lower, next to the doorframe. you save 5 energy for the jump and 1 for diagonal, but 1 less excess frame. I could clarify it as a devnote